baseplate/frameworks/queue_consumer/kafka.py

```diff
@@ -167,6 +168,69 @@ def handle(self, message: confluent_kafka.Message) -> None:
             raise
 
 
+def validate_group_id(name: str, group_id: str) -> None:
+    service_name, _, group_name = group_id.partition(".")
+    assert service_name and group_name, "group_id must start with 'SERVICENAME.'"
+    assert name == f"kafka_consumer.{group_name}"
+
+
+def make_kafka_consumer(
+    bootstrap_servers: str,
+    group_id: str,
+    topics: Sequence[str],
+    schema_registry: Optional[str] = None,
+    consumer_config: Dict[str, Any] = {},
```

Comment:
probably don't want a mutable default argument https://docs.python-guide.org/writing/gotchas/

Reply:
oh, yeah, i've read that and forgot. i was matching changes in thing, but I can have them be much more discrete. will move this back into the class and then don't need to pass in.